### PR TITLE
[fixes]multiple address fixes

### PR DESCRIPTION
**Pull request by saurabh6790**

### CODE REVIEW

File: erpnext_shopify/sync_customers.py [hunk 1]
```diff
@@ -56,11 +58,20 @@ def create_customer_address(customer, shopify_customer):
 			"customer_name":  customer.customer_name
 		}).insert()
 
+def get_address_title_and_type(customer_name, index):
+	address_type = _("Billing")
```
Comment (anandpdoshi) [inline erpnext_shopify/sync_customers.py]: How will this handle multiple Customer addresses?

Comment (anandpdoshi) [inline erpnext_shopify/sync_customers.py:65]: Doesn't shopify have "Shipping" type address?

Comment (saurabh6790) [inline erpnext_shopify/sync_customers.py:65]: @anandpdoshi 

Address type not exists in shopify.

Shopify Address structure is like : 

```
{
    "addresses": [
        {
            "id": 2514234118,
            "first_name": "s",
            "last_name": "p",
            "company": "",
            "address1": "Add5",
            "address2": "Add5",
            "city": "D",
            "province": "Andaman and Nicobar",
            "country": "India",
            "zip": "222222",
            "phone": "",
            "name": "s p",
            "province_code": "AN",
            "country_code": "IN",
            "country_name": "India",
            "default": true
        },
        {
            "id": 2514247366,
            "first_name": "s p",
            "last_name": "p",
            "company": null,
            "address1": "Address 1",
            "address2": "",
            "city": "City",
            "province": "Andaman and Nicobar",
            "country": "India",
            "zip": "",
            "phone": null,
            "name": "s p p",
            "province_code": "AN",
            "country_code": "IN",
            "country_name": "India",
            "default": false
        }
    ]
}
```

File: erpnext_shopify/sync_customers.py [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-def sync_shopify_customers():
+def sync_shopify_customers(shopify_customer_list):
 	for shopify_customer in get_shopify_customers():
 		if not frappe.db.get_value("Customer", {"shopify_customer_id": shopify_customer.get('id')}, "name"):
-			create_customer(shopify_customer)
+			create_customer(shopify_customer, shopify_customer_list)
 
-def create_customer(shopify_customer):
+def create_customer(shopify_customer, shopify_customer_list):
```
Comment (anandpdoshi) [inline erpnext_shopify/sync_customers.py:16]: `shopify_customer_list` not used

**Comment by saurabh6790** [comment]
@anandpdoshi Fixed.

**Comment by anandpdoshi** [comment]
merged manually into develop

**Pull request closed without merging**